draft-ietf-quic-transport.md

```diff
@@ -2765,6 +2765,9 @@ After sending a CONNECTION_CLOSE frame, an endpoint immediately enters the
 closing state; see {{closing}}. After receiving a CONNECTION_CLOSE frame,
 endpoints enter the draining state; see {{draining}}.
 
+Violations of the protocol lead to an immediate close, and a transport-level
+error being sent in a CONNECTION_CLOSE frame.
```

Comment:
```suggestion
Violations of the protocol lead to an immediate close, with a transport-level
error being sent in the CONNECTION_CLOSE frame.
```
I'm not sure ", and" is the right conjunction here; it sounds like two outcomes.